packages/pg-packet-stream/src/BufferReader.ts

```diff
@@ -0,0 +1,44 @@
+const emptyBuffer = Buffer.allocUnsafe(0);
```

Comment:
Is it intentional that the same Buffer instance be shared by all BufferReader instances?

Reply:
yeah it's just a placeholder to keep typescript happy & reduce allocations since it's never used.  It gets overwritten with `BufferReader.setBuffer` as soon as the buffer reader starts to be used.  Since you can't resize a buffer, an `allocUnsafe(0)` is fine to share - it's a length of 0 so you can't put anything into it, and you can't resize it.  I'll reduce the noise a bit since it is confusing. Thanks for the feedback!

Reply:
Thank you for the explanation!